photon-ml/src/test/scala/com/linkedin/photon/ml/optimization/LinearRegressionOptimizationProblemTest.scala

```diff
@@ -63,15 +63,15 @@ class LinearRegressionOptimizationProblemTest {
   def testComputeVariances(): Unit = {
     val problem = createProblem
     val input = mock(classOf[RDD[LabeledPoint]])
-    val coefficients = generateDenseVector(Dimensions)
+    val coefficients = generateDenseVector(DIMENSIONS)
 
     // TODO: computeVarainces is currently disabled. This test will need to be updated when the default changes
     assertEquals(problem.computeVariances(input, coefficients), None)
   }
 }
 
 object LinearRegressionOptimizationProblemTest {
-  val Dimensions = 10
+  val DIMENSIONS = 10
```

Comment:
I know it's inconsistent with how we've done things here, but the Scala style guide actually recommends upper camel case for constants:

[http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods](http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods)

I've been trying to nudge us in that direction, but we should probably make a wholesale change, or just have them all use the Java convention.